test/models/signerListSet.ts

```diff
@@ -44,6 +45,7 @@ describe('SignerListSet', function () {
 
   it(`verifies valid SignerListSet`, function () {
     assert.doesNotThrow(() => verifySignerListSet(SignerListSetTx))
+    assert.doesNotThrow(() => verify(SignerListSetTx))
```

Comment:
this should be camelCase: 
```suggestion
    assert.doesNotThrow(() => verify(signerListSetTx))
```